### PR TITLE
Pencil v3 added

### PR DESCRIPTION
**Pull request by loadaverage**
The Pencil Project's unique mission is to build a free and opensource tool for making diagrams and GUI prototyping that everyone can use.
https://github.com/evolus/pencil

### CODE REVIEW

**Comment by jessfraz** [comment]
cool I will try it out this weekend

**Comment by loadaverage** [comment]
@jessfraz thanks, I also added Freemind (https://github.com/jessfraz/dockerfiles/pull/238)

File: pencil/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.sw*
```
Comment (koep) [inline pencil/.gitignore]: I think this is covered by the global .gitignore in this repo

File: pencil/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+![Pencil](https://cldup.com/63hb_S0bDS.png)
```
Comment (koep) [inline pencil/README.md]: The "documentation" in this repo is usually represented by comments in the Dockerfiles (which you already did) This readme may be redundant and cause a small "maintenance overhead" don't you think?

File: pencil/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+```   
+**or**  
+- Pull image from Docker Registry  
+`docker pull loadaverage/pencil`
+- Allow access to X server (for some Linux distros)  
+`xhost local:pencil`
+- Run own Pencil image
+
+ ```bash
+    docker run --rm \
```
Comment (koep) [inline pencil/README.md]: I think you forgot to mention the X11 socket here.

**Comment by koep** [comment]
Oh and btw the image is pretty big:
```
$ docker images
REPOSITORY                                                                                    TAG                 IMAGE ID            CREATED             SIZE
pencil                                                                                        latest              c1ac211efbf6        17 minutes ago      1.02 GB
``` 
Any idea how to reduce the size?

**Comment by loadaverage** [comment]
@koep
sure, no problem! I can remove all the mess. 
Just spend a lot of time Dockerized Pencil and decided to add it here.
So, if you could help me a bit with the example of right documentation for this repo I will appreciate that.

**Comment by loadaverage** [comment]
> Any idea how to reduce the size?
@koep 
I dunno. The only idea - remove .git directory (~56M)

**Comment by koep** [comment]
Maybe you could base this image off the official node alpine image?
Its about 60 MB (compared to 600 MB). https://hub.docker.com/_/node/

**Comment by jessfraz** [comment]
Hi sorry, I try to keep these all maintained and only add ones I personally use so I am going to close this but your dockerfile is great and you should make a repo to house them for yourself :)

**Pull request closed without merging**